### PR TITLE
Cpu physical cores

### LINKED ISSUE #10: Wrap basic queries into variables like CPU_PHYSICAL_CORES
I'd say many people look into this package for a simple query of the number of cores, PUs, etc.

We should have these saved or wrapped into constants like `CPU_PHYSICAL_CORES` a la `Sys.CPU_CORES`. I will submit a PR when I find time.

### PR DESCRIPTION
**Pull request by juliohm**
Fix #10.

This is a simple utility for the most commonly used query. I appreciate if you can review it.

I thought of saving the result in a constant CPU_PHYSICAL_CORES but I am not sure it is safe to assume the number will be the same across a cluster of computers for example.

### CODE REVIEW

**Review by eschnett — CHANGES_REQUESTED** [review]
I would rename the function to `num_physical_cores`. I find the prefix `cpu` a bit confusing. (Is it the number of cores of one CPU only? No, it's the number of cores in a whole node.)

**Comment by juliohm** [comment]
Makes total sense @eschnett , thanks. I have pushed the changes and added an extra test for it.

**Review by eschnett — APPROVED** [review]

**Comment by GregPlowman** [comment]
Is it possible to tag a new release to incorporate this utility function?

**Comment by juliohm** [comment]
@eschnett I thought we had tagged a new version after this PR was merged, could you please take care of it?

**Comment by eschnett** [comment]
@juliohm I don't recall. I believe you have write access to the repository -- can you tag a new version yourself?

**Comment by juliohm** [comment]
Thank you @eschnett, I tagged a new version.

**Pull request merged**
